proposals/0019-mesh-nodes.md

```diff
@@ -275,6 +275,13 @@ Some restrictions on the function use and interactions with output arrays follow
 See [SetMeshOutputCounts](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/MeshShader.md#setmeshoutputcounts)
 in the mesh shader spec for specific details, restrictions, and examples.
 
+### Derivative Operations
+Because mesh shaders and node shaders with the broadcasting launch mode both support
```

Comment:
Nit (markdown lint): use one blank line after heading, and only one blank line before next heading